components/process.rst

```diff
@@ -139,6 +139,21 @@ environment variables using the second argument of the ``run()``,
     // On both Unix-like and Windows
     $process->run(null, ['MESSAGE' => 'Something to output']);
 
+To help write command lines that are independent from the operating system,
+you can also write the above code as such::
```

Comment:
```suggestion
you can also write the above code as a portable command::
```

Otherwise, when removing the versionadded in, nobody finds this by searching for portable commands. Or is this not necessary @nicolas-grekas. We can do the change while merging, no rework from your side needed, if.